examples/sycl/06_bmg_flash_attention/bmg_flash_attn_prefill_cachedKV_runner.hpp

```diff
@@ -86,6 +87,16 @@ struct Options {
       varlen = true;
     }
 
+    if (cmd.check_cmd_line_flag("use_paged_kv")) {
```

Comment:
Since `paged_kv` is off by default, could you add to `examples/sycl/06_bmg_flash_attention/CMakeLists.txt` to make the CI run both options, something like:

```
set(TEST_NO_PAGED "")
set(TEST_PAGED "--use_paged_kv")
```

and then modify the add_executable as follows:
```
cutlass_example_add_executable(
  06_bmg_prefill_attention_cachedKV
  06_bmg_prefill_attention_prefill_cachedKV.cpp
  TEST_COMMAND_OPTIONS
  TEST_NO_PAGED
  TEST_PAGED
)
```